pkg/v1/tkg/test/tkgctl/docker/docker_ceip_test.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2021 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// nolint:typecheck,nolintlint
```

Comment:
(question): why do we need these disabled?

Reply:
had the same question.

Reply:
I just added this file similar to other test files under tkgctl/docker added by the CLI team. I'll check what would happen with these removed and will let you know in a bit